src/content/guides/compatible-solutions.mdoc

```diff
@@ -17,6 +17,8 @@ Centrapay is currently integrated with a wide range of payment and point-of-sale
 - G6-400
 - G6-500
 
+### Ontempo
```

Comment:
Are they a POS or online solution? Might be worth adding some detail in the bullet point